src/components/RadioButton.js

```diff
@@ -7,13 +7,13 @@ export const RadioButton = (props) => {
     fontSize: 15,
     lineHeight: '2rem',
     color: 333,
-    fontFamily: '\'Roboto\', sans-serif',
+    fontFamily: props.font ? props.font : '\'Roboto\', sans-serif',
```

Comment:
This approach is very similar to the previous approach. To make RadioButton more flexible, allow it to take in a style object that can be merged into its own style. That way it is generic. With the current approach, we'd need to add new props for each style prop that might need to change. With the object approach, it is flexible enough to handle any new style without the need for modification.